### PR TITLE
feat: add tests for `MsgCreatePool` Tx

### PR DESCRIPTION
**Pull request by hansol-medi**
- add `wasmkeeper` to test suite for deployment of NFT contract in test
- tests for `MsgCreatePool` Tx have been added
  - `pool_keeper_test`
  - `grpc_query_pool_test`
  - `genesis_test`
- add `nft_contract_address` to genesis params

### CODE REVIEW

File: proto/panacea/datapool/v2/pool.proto [hunk 1]
```diff
@@ -54,7 +54,11 @@ message PoolParams {
   repeated string          data_schema = 1;             // JSON Schema URIs
   uint64                   target_num_data = 2;         // A target num of data to be collected
   uint64                   max_nft_supply = 3;          // A max num of NFTs that can be minted (= A total num of potential buyers)
-  cosmos.base.v1beta1.Coin nft_price = 4;               // A price of a NFT
```
Comment (gyuguen) [inline proto/panacea/datapool/v2/pool.proto:57]: Is there any reason to change to accept multiple coins?

**Review by gyuguen — APPROVED** [review]
LGTM 👍 
I left one question :)

File: proto/panacea/datapool/v2/pool.proto (hunk 1 above)
Comment (hansol-medi) [reply proto/panacea/datapool/v2/pool.proto:57]: Actually, I don't remember why I changed the type of `nft_price` to sdk.Coins. So I do roll back it

File: x/datapool/genesis.go [hunk 2]
```diff
@@ -12,6 +12,10 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 	k.SetParams(ctx, genState.Params)
 	k.SetPoolNumber(ctx, genState.NextPoolNumber)
 
+	if genState.NftContractAddress != nil {
+		k.SetNFTContractAddress(ctx, genState.NftContractAddress)
+	}
```
Comment (youngjoon-lee) [inline x/datapool/genesis.go:17]: I think it's okay to remove this `if` statement, even though the `DefaultGenesis.NftContractAddress` is `nil`: https://github.com/medibloc/panacea-core/pull/295/files#diff-1b224410c503ee7f657889a5370b3ff80d2396445a85534d2e4a544bbe357aecR12.

Comment (youngjoon-lee) [reply x/datapool/genesis.go:17]: Ah, I understood why you added this `if` statement. 
https://github.com/medibloc/panacea-core/blob/722287ea7d0b093a1a3517a54310791f5bdab032/x/datapool/keeper/pool.go#L253

Okay then, let's go as it is.

**Review by youngjoon-lee — APPROVED** [review]
lgtm

**Review by inchori — APPROVED** [review]
LGTM 😃

**Pull request merged**